compiler/luci/pass/include/luci/Pass/ConvertNCHWToNHWCPass.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_CONVERT_NCHW_TO_NHWC_PASS_H__
+#define __LUCI_CONVERT_NCHW_TO_NHWC_PASS_H__
+
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief  Class to convert NCHW Ops to NHWC
+ * @details Find operators that use NCHW layout and make them use NHWC.
```

Comment:
Minor thing, can you please (1) align `Class` and `Find` (2) add an empty line between @brief and @details

Reply:
Hm... `@brief` is a valid user account... and also `@details`... sorry for sending notifications :)

Reply:
> sorry for sending notifications :)

No problem :)

Updated comments. Thanks.